pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1210,6 +1210,66 @@ func (configgen *ConfigGeneratorImpl) generateManagementListeners(node *model.Pr
 	return listeners
 }
 
+// buildSidecarVirtualOutboundListener creates the outbound virtual listener
+// which receives traffic for IPTABLES redirect
+func (configgen *ConfigGeneratorImpl) buildSidecarVirtualOutboundListener(env *model.Environment,
```

Comment:
can you rename this to something else please? This is not really building but calling plugins. And its confusing given that the listener builder also has a buildVirtrual...  may be onVirtualOutboundListener() ?

Reply:
That's fair, will update it.

Reply:
Done